src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -64,8 +62,6 @@
  *  persistent connections. Would be great to implement
  *  this feature. BkBasic.accept should handle more
  *  than one HTTP request in one connection.
- * @todo #516:30min It will be nice to refactor tests with Socket usage and
- *  replace them to real statements. See usage of BkBasicTest.createMockSocket.
  * @todo #516:15min Move header names from BkBasic to public constants.
```

Comment:
@essobedo you had to use 533 here instead of 516
that's why #516 not closed yet I guess